test/src/test/java/test/org/dst/rpc/DstRpcServerTest.java

```diff
@@ -33,19 +35,21 @@ public void testRpcServer() {
     // Test string put request
     DstServerProtocol.StringPutRequest stringPutRequest =
             DstServerProtocol.StringPutRequest.newBuilder()
-        .setKey("k1")
-        .setValue("v1")
-        .build();
+            .setKey("k1")
```

Comment:
Why change these lines?

Reply:
`.setKey()` should be aligned with the code above?

Reply:
But current CI doesn't complain about anything.
It means the code style is fine before, right?